include/z64.h

```diff
@@ -1403,7 +1403,7 @@ typedef enum {
     /* 16 */ F_B8
 } FloorID;
 
-// All arrays pointed in this struct are indexed by "map indexes"
+// All arrays pointed in this struct are indexed by "map indices"
```

Comment:
I'm in the indexes team, but that's just personal preference and I understand why we would favour indices instead

Reply:
as explained in the commit message I went with "indices" because it currently prevails, and seems vaguely more appropriate with respect to common usage, but we could also change "indices" to "indexes" instead (I would be personally defending "indices")